extensions/libraries/joomla_entity/src/Fields/Traits/HasFields.php

```diff
@@ -54,7 +69,10 @@ public function field($id)
 			throw new \InvalidArgumentException($msg);
 		}
 
-		return $fields->get($id);
+		/** @var Field $field */
+		$field = $fields->get($id);
```

Comment:
This just adds lines of code for nothing?

Reply:
Just to please type checker, because the `get` function returns an `EntityInterface` and the return value is `Field`